cmd/soroban-cli/src/commands/contract/deploy.rs

```diff
@@ -22,6 +22,8 @@ use crate::{
     utils, wasm,
 };
 
+const CONTRACT_META_SDK_KEY: &str = "rssdkver";
```

Comment:
Is this from the sdk? If so could we import it?

Reply:
I found this [contractmetabuiltin macro](https://github.com/stellar/rs-soroban-sdk/blob/main/soroban-sdk-macros/src/lib.rs#L267) but it doesn't look like we can import the contract meta key for SDK version in a straightforward way.